src/meta/src/manager/catalog/database.rs

```diff
@@ -185,6 +199,18 @@ impl DatabaseManager {
         }
     }
 
+    pub fn check_connection_name_duplicated(&self, relation_key: &RelationKey) -> MetaResult<()> {
```

Comment:
Yes, we should only check if it's duplicate in the connection collection. But in frontend it's checked among all relations.